### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by siddharth9820**

### CODE REVIEW

File: .github/workflows/formatting-tests.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: unit tests
+name: code-format tests
```
Comment (bhatele) [inline .github/workflows/formatting-tests.yaml]: We can call this just "formatting tests"

File: .github/workflows/formatting-tests.yaml [hunk 2]
```diff
@@ -27,7 +27,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
     - name: Update black
       if: ${{ matrix.python-version == 3.9 }}
```
Comment (bhatele) [inline .github/workflows/formatting-tests.yaml]: these tests seem to be running for multiple python versions right? which seems unnecessary

File: .github/workflows/mordor-tests.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+name: mordor tests
```
Comment (bhatele) [inline .github/workflows/mordor-tests.yaml]: we don't need to make the machine name public.
Both the filename and test can be called something like nvidia-<gpumodel> tests

File: axonn/optim.py [hunk 4]
```diff
@@ -15,7 +15,7 @@ def __init__(
         lr=1e-3,
```
Comment (bhatele) [inline axonn/optim.py:20]: this file should probably be called optimizer.py and we should also add an __init__.py

File: examples/test_vit.py [hunk 5]
```diff
@@ -4,27 +4,32 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```
Comment (bhatele) [inline examples/test_vit.py:4]: If this is the test that is run, it should be under axonn/tests/

File: axonn/optim.py (hunk 4 above)
Comment (siddharth9820) [reply axonn/optim.py:20]: Pytorch serves its optimizer as ``torch.optim.<optimizer-name>``.
I thought we can do the same - ``axonn.optim.<>``

File: tests/test_vit.py [hunk 6]
```diff
@@ -4,27 +4,32 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```
Comment (bhatele) [inline tests/test_vit.py]: tests directory should go under axonn/axonn

File: .github/workflows/nvidia-tests.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+name: nvidia tests
```
Comment (bhatele) [inline .github/workflows/nvidia-tests.yml]: filename change to .yaml to be consistent

File: .github/workflows/formatting-tests.yaml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-name: unit tests
+name: formatting tests
 
 on:
   push:
```
Comment (bhatele) [inline .github/workflows/formatting-tests.yaml:4]: lets rename the job name from build to formatting

**Pull request merged**